pkg/reconciler/buildrun/resources/runtime_image.go

```diff
@@ -112,6 +112,7 @@ func renderEntrypoint(e []string) string {
 
 // renderRuntimeDockerfile render runtime Dockerfile using build instance and pre-defined template.
 func renderRuntimeDockerfile(b *buildv1alpha1.Build) (*bytes.Buffer, error) {
+
```

Comment:
Only change in this file and it it not changing anything. I think we should drop it.
```suggestion
```